app/mixins/event-wizard.js

```diff
@@ -48,7 +48,9 @@ export default Mixin.create(MutableArray, CustomFormMixin, {
   async saveEventData(propsToSave = []) {
     const event = this.get('model.event');
     const data = {};
-
+    if (!propsToSave.includes('tickets')) {
+      propsToSave.push('tickets');
```

Comment:
instead of this, just add it where the function is being called.

Reply:
@CosmicCoder96 Followed the suggestion, please review now